wiki/Installation-guide.md

```diff
@@ -22,8 +22,8 @@ For any problem visit the [Troubleshooting page](https://github.com/alvr-org/ALV
 
 ### Windows microphone streaming
 
-To use the microphone you need to install the [VB-CABLE driver](https://vb-audio.com/Cable/).
-Set "CABLE Output" as the default microphone.
+To use the microphone you need to install the [VB-CABLE driver](https://vb-audio.com/Cable).
+Set "VB Cable" as the default microphone.
```

Comment:
If you change this in the wiki it should also be changed in code. There's also the question of how to handle old and new sink names for VB Cable

Reply:
The old option will not be recognized in the session and it will be lost, but session extrapolation will preserve all the other settings. This is a compromise I decided when defining protocol compatibility